cmake/modules/AddSwift.cmake

```diff
@@ -1384,6 +1389,15 @@ function(add_swift_library name)
           endif()
         endforeach()
 
+        set(swiftlib_framework_depends_flattened ${SWIFTLIB_FRAMEWORK_DEPENDS})
```

Comment:
Sorry, yes there's a move.  I still had the 'git diff' my local machine shows in my head.  In it, it appears that the loop over swiftlib_framework_depends_flattened moves, which it didn't really.  What actually moved was the content I described above.